server/routes/transaction.routes.js

```diff
@@ -142,4 +145,35 @@ router.get(
   TransactionController.getOneTransaction,
 );
 
+/**
+ * @swagger
+ * /transaction/remittance:
+ *   put:
+ *     tags:
+ *       - Transaction
+ *     description: Adds remittance for a specific transaction
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/Transaction'
+ *      description: Adds remittance for a specific transaction
+ *     responses:
+ *      '200':
+ *        description: Remittance added
+ *      '400':
+ *        description: Error adding remittance
+ *      '500':
+ *       description: Internal server error
+ */
+router.put(
```

Comment:
Can we change this to `post` since we are using the endpoint to add a new remittance 🙏🏽 . This makes the endpoint consistent even if we need to move the remittance model into its own model. It also saves us the stress if we wish to create an endpoint to update remittance. 

I know it is confusing but I feel post is the right thing to use here even though we are updating the transaction model.

```suggestion
router.post(
```